keps/sig-node/0009-node-heartbeat.md

```diff
@@ -243,9 +255,19 @@ The changes in components logic (Kubelet, NodeController) should be done behind
 a feature gate. We suggest making that enabled by default once the feature is
 implemented.
 
+Beta:
+- Proved scalability/performance gain. We expect significant decrease in total
```

Comment:
Please note that this has been concerned (presumably via kubemark?).

Reply:
sorry: confirmed

Reply:
Both kubemark but also real clusters - also clarified in the text.